governance/.env

```diff
@@ -0,0 +1,27 @@
+# private key of the signer that will deploy the contract
+export DEPLOYER_PRIVATE_KEY="0x14e1a3103ce06d0348acf19df3df1965ee55f9cdb16708c0bb8cd3d843ab8805"
+
+# block explorers API keys
+export ETHERSCAN_API_KEY="988a82bfe0a540ad84aa1a5d534c2bc6"
+export POLYGONSCAN_API_KEY=
+export OPTIMISTIC_ETHERSCAN_API_KEY=
+export BSCSCAN_API_KEY=
+export SNOWTRACE_API_KEY=
+export ARBISCAN_API_KEY=
+export CELO_API_KEY=
+export GNOSISSCAN_API_KEY=
+
+# This is a sample .env file for use in local development.
+
+# Duplicate this file as .env
+
+# Your Private key
+DEVNET_PRIVKEY="0x14e1a3103ce06d0348acf19df3df1965ee55f9cdb16708c0bb8cd3d843ab8805"
```

Comment:
You should never check a private key into a git repo...

Reply:
Yeah totally agree with that, I didn't know .env wasn't included in the .gitignore file, especially because I saw a .env.copy file already present in the ```/governance/``` folder, which I copied and renamed to .env. For what it's worth it's a fresh wallet I created specifically for the tests.